proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -424,8 +444,26 @@ message ReceiveComponentDataStreamResponse {
 
 // Request parameters for the RPC `ReceiveSensorDataStream`.
 message ReceiveSensorDataStreamRequest {
+  // A message for specifying a filter to apply to the stream.
+  message SensorDataStreamFilter {
+    // List of metrics to return. Only the specified metrics will be returned.
+    // A complete list of metrics can be found [here](https://github.com/frequenz-floss/frequenz-api-common/blob/v0.5.0/proto/frequenz/api/common/v1/metrics/metric_sample.proto#L65-L66).
```

Comment:
I would not put in the link here. The import path is already given and also line 455 gives the entire path where to find it. This is too much.

Reply:
Done.